com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -16,6 +16,11 @@ public sealed class NetworkObject : MonoBehaviour
         [SerializeField]
         internal uint GlobalObjectIdHash;
 
+        /// <summary>
+        /// For in-scene placed NetworkObjects, this is used to uniquely identify the
+        /// NetworkObject relative to the NetworkSceneHandle.
+        /// </summary>
+        internal int NetworkSceneHandle;
```

Comment:
this doesn't seem like a innocent trivial change to me 👀

Reply:
I think I can remove that... glad you called it out.